homeassistant/components/withings/common.py

```diff
@@ -636,7 +636,7 @@ def async_stop_polling_webhook_subscriptions(self) -> None:
             self._cancel_subscription_update()
             self._cancel_subscription_update = None
 
-    async def _do_retry(self, func, attempts=3) -> Any:
+    async def _do_retry(self, func, attempts=5) -> Any:
```

Comment:
In one of tests locally I've seen request succeeding on 4th call - it was only one time, and might be caused by something else I was doing during investigation. 

It would probably work with three retries now when the race condition is fixed - but I think then I'd need to increase the back-off time also as we would need to have last retry at least one second after first. Maybe having back-off of 0.5 seconds per attempt? WDYT?

There is another related improvement I'm looking into that would reduce need for retries - right now when it fails during token refresh for almost any reason (throttle, backend timeoutetc), we have authentication error and need to retry, but in reality the sensor should probably just go offline until next webhook call or next update tick, and we should get re-auth request only when withings tell us that. But that fix might have more side effects, so I didn't want to include it here.

Reply:
I'd probably increase the time between retries a bit as you mentioned. But you should decide since you're testing and know what works and not.

Reply:
I'll go then with three retries and longer times between.